Makefile

```diff
@@ -179,6 +182,13 @@ $(DIST)/add-on/chrome/content/preferences/prefs.js: $(VENV) add-on/preferences/p
 	  python $(VENV)/lib/python2.7/site-packages/mozbuild/preprocessor.py \
 	         -D LOOP_BETA=1 -o $@ add-on/preferences/prefs.js
 
+$(BUILT)/add-on/install.rdf: $(VENV) Makefile add-on/install.rdf.in
+	@mkdir -p $(@D)
+	. $(VENV)/bin/activate && \
+	  python $(VENV)/lib/python2.7/site-packages/mozbuild/preprocessor.py \
+	         -D MOZ_APP_MAXVERSION=$(MOZ_APP_MAXVERSION) \
+	         -o $@ add-on/install.rdf.in
```

Comment:
I decided to go for it. I think its reasonable either way, but $< cuts the repetition.